package.json

```diff
@@ -1,16 +1,9 @@
 {
-  "name": "@editorjs/editorjs",
+  "name": "@discolearn/editorjs",
   "version": "2.27.0-rc.1",
-  "description": "Editor.js — Native JS, based on API and Open Source",
+  "description": "Disco patch of Editor.js — Native JS, based on API and Open Source",
```

Comment:
Not sure what info should be updated/removed in here, took a first stab at it. 

Following the suggestion in [this thread](https://github.com/semver/semver/issues/17#issuecomment-49895206), I think it would be correct to leave the version here as is, and in my other branch update to: `2.27.0-rc.1+0.0.1` ?

Very open to feedback here.

Reply:
Testing something, will [this](https://togithub.com/discolearn/editor.js/pull/1) reference my PR?